dist/main/index.js

```diff
@@ -103325,6 +103331,20 @@ async function setupBazelisk() {
   core.endGroup()
 }
 
+async function isBazelAvailable() {
+  try {
+    await exec.exec('bazelisk', ['version'], { silent: true })
+    return true
+  } catch (error) {
+    try {
+      await exec.exec('bazel', ['version'], { silent: true })
```

Comment:
I might be picky, but this would actually start downloading the bazel and extracting it. Maybe we should instead use `which bazel` and Windows-equivalent to avoid expensive download/extract operations.